### PR TITLE
Update ome-faq.yml

### PR DESCRIPTION
**Pull request by samschan-ms**
Updated delegate access doc

@KCCross please review and merge

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/ome-faq.yml [hunk 1]
```diff
@@ -243,11 +238,11 @@ sections:
              ```powershell
              Add-MailboxPermission -Identity support@contoso.onmicrosoft.com -User ayla@contoso.com -AccessRights FullAccess -AutoMapping $true
              ```
-          
+
       - question: |
-          Can I open encrypted messages sent to another user's mailbox with Fullaccess?
+          Is delegated access supported with opening encrypted messages? Even if delegate has Fullaccess on another user's mailbox?
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          Is delegated access supported with opening encrypted messages? Even if a delegate has full access to another user's mailbox?
```

File: microsoft-365/compliance/ome-faq.yml [hunk 2]
```diff
@@ -243,11 +238,11 @@ sections:
...
         answer: |
-          No. You can only open encrypted messages in a shared mailbox.
+          Delegate access of encrypted mail is supported in Outlook on the web, Outlook for Mac, Outlook for iOS, and Outlook for Android.  Outlook for Windows do not support delegate access.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          Delegated access of encrypted mail is supported in Outlook on the web, Outlook for Mac, Outlook for iOS, and Outlook for Android.  Outlook for Windows does not support delegated access.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @samschan-ms, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Pull request merged**